spec/system/admin/orders_spec.rb

```diff
@@ -497,8 +497,9 @@
         expect(page.find(
                  "#listing_orders tbody tr td:first-child input[type=checkbox]"
                )).to_not be_checked
-        # disables print invoices button
-        page.find("span.icon-reorder", text: "ACTIONS").click
+        # disables print invoices button not clickable
+        expect { find("span.icon-reorder", text: "ACTIONS").click }
+          .to raise_error(Capybara::Cuprite::MouseEventFailed)
```

Comment:
Couldn't we check for the presence of a css class ? I'd assume we use a `disabled` class or similar to make the print button not clickable.

Reply:
Hello @rioug , 

I find it better to test it that way because we are sure the button is not clickable and that no menu show up.

The details element haven't got a native `disabled` state, contrary to input elements.
So a `disabled` defined class is not enough as there is no way to be sure it enforces a non clickable behaviour.
Actually, I had to add a `pointer-events: none` in the CSS to enforce the non clickable behaviour.
The previous disabled class did not prevent the clickable behaviour on a details as it was designed for a div.

Reply:
Fair enough, thanks for the explanation !